src/main/java/employeetracker/model/employee/UniqueEmployeeList.java

```diff
@@ -293,16 +301,16 @@ public String getShortestTenure() {
     }
 
     /**
-     * Get the Average Tenure for all employee in the list.
+     * Gets the Average Tenure for all employee in the list.
      */
-    public String getAvgTenure() {
+    public String getAverageTenure() {
         Date todaysDate = new Date();
         Date dateOfJoin;
         long diffInMillies;
         long yearMiniSec = Long.parseLong("31536000000");
         long dayMiniSec = Long.parseLong("86400000");
         long totalMillies = 0;
-        long avgTensure = 0;
+        long averageTensure = 0;
```

Comment:
Perhaps it should be `averageTenure ` instead of `averageTensure`? Tensure is not an english word